### PR TITLE
translate code-editors page

### PR DESCRIPTION
**Pull request by jamal-pb95**

### CODE REVIEW

**Comment by jamal-pb95** [comment]
@raikusy @sjsakib looking for your review guys

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-A code editor is the place where programmers spend most of their time.
+একজন প্রোগ্রামার তার জীবনের বড় একটা সময় কাঁটায় কোড এডিটরের সাথে।
 
-There are two main types of code editors: IDEs and lightweight editors. Many people use one tool of each type.
+প্রধানত দুই প্রকারের কোড এডিটর রয়েছে। IDE ও লাইটওয়েট এডিটর সমূহ। প্রায় অনেকেই একটি IDE ও একটি লাইটওয়েট এডিটর ব্যবহার করে থাকেন।
 
 ## IDE
 
-The term [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) (Integrated Development Environment) refers to a powerful editor with many features that usually operates on a "whole project." As the name suggests, it's not just an editor, but a full-scale "development environment."
+[IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) বলতে মুলত বুঝায় যে - (একটি সমন্বিত ডেভেলপমেন্ট পরিবেশ) যার সাথে একটি পাওয়ারফুল এডিটরও থাকে, যাতে অনেক অনেক কার্যকারী ফিচারসমূহ থাকে। যেগুলো একজন ডেভেলপারের জন্য একটি পূর্ণ প্রোজেক্ট সম্পূর্ণ করতে অনেক সহায়তা করে। নাম থেকে যা বুঝা যায় তা হল এটি শুধুমাত্র একটি এডিটর নয়, বরং একটি প্রোজেক্ট করার জন্য ফুল-স্কেল ডেভেলপমেন্ট পরিবেশ।
```
Comment (raikusy) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
[IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) বলতে মুলত বুঝায় যে - (একটি সমন্বিত ডেভেলপমেন্ট পরিবেশ) যার সাথে একটি পাওয়ারফুল এডিটরও থাকে, যাতে অনেক অনেক কার্যকারী ফিচারসমূহ থাকে। যেগুলো একজন ডেভেলপারের জন্য একটি পূর্ণ প্রোজেক্ট সম্পূর্ণ করতে অনেক সহায়তা করে। নাম থেকে যা বুঝা যায় তা হল এটি শুধুমাত্র একটি এডিটর নয়, বরং একটি প্রোজেক্ট করার জন্য স্বয়ংসম্পূর্ণ ডেভেলপমেন্ট পরিবেশ।
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+বেশির ভাগ IDE সমূহ পেইড, তবে তাদের প্রায় সবার ট্রায়াল ভার্শন রয়েছে। এসবের মূল্যও একজন কোয়ালিফাইড ডেভেলপারের বেতনের তুলনায় খুবই নগণ্য। সুতরাং আপনার যেটা ভালো লাগে সেটা পছন্দ করতে পারেন।
 
-## Lightweight editors
+## লাইটওয়েট এডিটর সমূহ
 
-"Lightweight editors" are not as powerful as IDEs, but they're fast, elegant and simple.
+"লাইটওয়েট এডিটর সমূহ" IDE গুলোর মত এতটা পাওয়ারফুল নয়। তবে অনেক ফাস্ট, এলিগেনট ও সিম্পল হয়ে থাকে।
 
-They are mainly used to open and edit a file instantly.
+প্রথানত দ্রুত কোন ফাইল ওপেন ও এডিট করার ক্ষেত্রে এসব এডিটর ব্যবহার করা হয়ে থাকে।
```
Comment (raikusy) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
প্রধানত দ্রুত কোন ফাইল ওপেন ও এডিট করার ক্ষেত্রে এসব এডিটর ব্যবহার করা হয়ে থাকে।
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
 
-The main difference between a "lightweight editor" and an "IDE" is that an IDE works on a project-level, so it loads much more data on start, analyzes the project structure if needed and so on. A lightweight editor is much faster if we need only one file.
+লাইটওয়েট এডিটর সমূহের সাথে IDE গুলোর প্রধান পার্থক্য হল যে, IDE গুলো ব্যবহার করা হয় প্রোজেক্ট লেভেলের কাজে। সুতরাং এগুলো শুরুতেই অনেক বেশি ডাটা লোড করে, প্রোজেক্টের গঠন পর্যবেক্ষণ করে ইত্যাদি। এদিক থেকে লাইটওয়েট এডিটর অনেক বেশি দ্রুততর হয়ে থাকে শুধুমাত্র একটি ফাইলে কাজ করার ক্ষেত্রে।
 
-In practice, lightweight editors may have a lot of plugins including directory-level syntax analyzers and autocompleters, so there's no strict border between a lightweight editor and an IDE.
+কাজের ক্ষেত্রে, লাইটওয়েট এডিটর সমূহের জন্য অনেক অনেক প্লাগিনস রয়েছে - ডিরেক্টরি লেভেল সিনট্যাক্স আনালাইজার থেকে শুরু করে অটোকমপ্লিট সহ নানান কাজ খুব সহজেই করা যায় এসব প্লাগিনস ব্যবহার করে। সুতরাং একটি লাইটওয়েট এডিটর ও IDE এর মধ্যে তেমন বিশাল কোন সীমারেখা নাই।
```
Comment (raikusy) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
কাজের ক্ষেত্রে, লাইটওয়েট এডিটর সমূহের জন্য অনেক অনেক প্লাগিনস রয়েছে - ডিরেক্টরি লেভেল সিনট্যাক্স এনালাইজার থেকে শুরু করে অটোকমপ্লিট সহ নানান কাজ খুব সহজেই করা যায় এসব প্লাগিনস ব্যবহার করে। সুতরাং একটি লাইটওয়েট এডিটর ও IDE এর মধ্যে তেমন বিশাল কোন সীমারেখা নাই।
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
-- [Atom](https://atom.io/) (cross-platform, free).
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [Sublime Text](http://www.sublimetext.com) (cross-platform, shareware).
-- [Notepad++](https://notepad-plus-plus.org/) (Windows, free).
-- [Vim](http://www.vim.org/) and [Emacs](https://www.gnu.org/software/emacs/) are also cool if you know how to use them.
+- [Atom](https://atom.io/) (এটি ক্রস প্ল্যাটফর্ম ও সম্পূর্ণ ফ্রি)।
+- [Visual Studio Code](https://code.visualstudio.com/) (এটি ক্রস প্ল্যাটফর্ম ও সম্পূর্ণ ফ্রি)।
+- [Sublime Text](http://www.sublimetext.com) (এটি ক্রস প্ল্যাটফর্ম ও কিছু ক্ষেত্রে ফ্রি)।
+- [Notepad++](https://notepad-plus-plus.org/) (উইন্ডোজ, ফ্রি)।
+- [Vim](http://www.vim.org/) এবং [Emacs](https://www.gnu.org/software/emacs/) আপনি যদি ঠিক ব্যবহার করতে পারেন তবে, এগুলোও অনেক কুল এডিটর।
```
Comment (raikusy) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
- [Vim](http://www.vim.org/) এবং [Emacs](https://www.gnu.org/software/emacs/) ও অসাধারণ এডিটর, যদি আপনি এদের সঠিক ব্যবহার করতে জানেন।
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
 
-## Let's not argue
+## চলুন ঝগড়া বাদ দিয়ে কিছু কথা বলিঃ
 
-The editors in the lists above are those that either I or my friends whom I consider good developers have been using for a long time and are happy with.
+উপরে উল্লেখিত এডিটর সমূহ আমি ও আমার বন্ধুরা মনে করি যে, ভালো ডেভেলপাররা অনেক দিন যাবত ব্যবহার করছে এবং তারা এগুলো নিয়ে সুখে আছে।
```
Comment (raikusy) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
উপরে উল্লেখিত এডিটর সমূহ আমি অথবা আমার বন্ধু যাদের আমি ভালো ডেভেলপার মনে করি, তারা অনেক দিন যাবৎ বেশ স্বাচ্ছন্দ্যে ব্যবহার করছে।
```

File: 1-js/01-getting-started/3-code-editors/article.md [hunk 6]
```diff
@@ -1,46 +1,46 @@
...
 
-There are other great editors in our big world. Please choose the one you like the most.
+আমাদের এই বৃহৎ পৃথিবীতে আরও অনেক ভালো ভালো এডিটর আছে। আপনি আপনার পছন্দ অনুযায়ী যেকোন একটা ব্যবহার করতে পারেন।
 
-The choice of an editor, like any other tool, is individual and depends on your projects, habits, and personal preferences.
+একটি এডিটর পছন্দ করা অনেক সময় আপনার প্রোজেক্ট, পূর্ব-অভিজ্ঞতা, কাজের সাচ্ছন্দ এসবের উপর নির্ভর করে।
```
Comment (raikusy) [inline 1-js/01-getting-started/3-code-editors/article.md]:
```suggestion
একটি এডিটর বাছাই করা অনেক সময় আপনার প্রোজেক্ট, পূর্ব-অভিজ্ঞতা, কাজের স্বাচ্ছন্দ্য এসবের উপর নির্ভর করে।
```

**Comment by jamal-pb95** [comment]
Thanks for your kind review 😊😘. I will submit my changes soon.

**Pull request merged**